plugins/DuskVerb/tools/tuner/fleet_audit.py

```
@@ -211,8 +211,17 @@ def render_and_check(name, no_render=False):
             shutil.copy(src, f"{lex}/anchor_{s}.wav")
     if not os.path.exists(f"{lex}/anchor_noiseburst.wav"):
         return f"NO_ANCHOR {name} {adir}", False
-    fc = subprocess.run([sys.executable, FC, dv, lex, "--name", name, "--json"],
-                        capture_output=True, text=True, timeout=200)
+    try:
+        fc = subprocess.run([sys.executable, FC, dv, lex, "--name", name, "--json"],
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)